x/oracle/module.go

```diff
@@ -164,8 +164,7 @@ func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Val
 		validatorList := make(map[string]*big.Int)
 		for _, vu := range validatorUpdates {
 			pubKey, _ := cryptocodec.FromTmProtoPublicKey(vu.PubKey)
-			validator, _ := am.keeper.GetValidatorByConsAddr(ctx, sdk.GetConsAddress(pubKey))
-			validatorList[validator.OperatorAddress] = big.NewInt(vu.Power)
+			validatorList[sdk.AccAddress(pubKey.Address()).String()] = big.NewInt(vu.Power)
```

Comment:
This is not the operators `AccAddress`. It is the address corresponding to the consensus key, converted to bytes, and then converted to `AccAddress`. For clarity, can we use `ConsAddress(pubKey.Address()).String()` instead?

Reply:
Yes, the requirement for this is to be consistent, either `AccAddress` or `ConsAddress` will suite this. The reason to choose `AccAddress` is that this is used to sign a transaction. But yes `ConsAddress` also make sense since it's derived from consensusKey

Reply:
Updated. use ConsAddress to describe address for consensus key